system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -40,6 +40,34 @@
                   :ContactPersons [{:Roles ["Data Center Contact"]
                                     :LastName "Smith"}]}]})
 
+(def data-centers-url
```

Comment:
Add a docstring and rename. This is not just a data center URL. Maybe `data-centers-with-urls`?

Reply:
Done. Changed it to data-centers-umm-with-home-page-url.